zulipterminal/ui_tools/views.py

```diff
@@ -558,13 +558,22 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             return key
 
         elif is_command_key('STREAM_MESSAGE', key):
+            if(self.model.is_user_in_unsubscribed_stream(
+               self.model.stream_id)):
+                self.model.controller.view.set_footer_text(
+                        " You can't write messages from unsubscribed streams.",
+                        3)
```

Comment:
This seems like a sufficiently duplicated check/string to have a general message of the type as indicated in another comment? Or even a helper method in the controller.